test/e2e/scenario/rum/resources.scenario.ts

```diff
@@ -201,6 +201,56 @@ describe('rum resources', () => {
         expect(resourceEvent?.resource.status_code).toBe(0)
       })
   })
+
+  describe('support XHRs with same XMLHttpRequest instance', () => {
+    createTest('track only second XHR (first is canceled by the browser)')
+      .withRum()
+      .withSetup(bundleSetup)
+      .run(async ({ events }) => {
+        await browserExecuteAsync((done) => {
+          const xhr = new XMLHttpRequest()
+          xhr.open('GET', '/ok?duration=200&call=1')
+          xhr.send()
+          xhr.open('GET', '/ok?duration=100&call=2')
+          xhr.send()
+          setTimeout(() => {
+            done(undefined)
+          }, 200)
+        })
+
+        await flushEvents()
+
+        const resourceEvents = events.rumResources.filter((event) => event.resource.type === 'xhr')
+        expect(resourceEvents.length).toEqual(1)
+        expect(events.rumErrors.length).toBe(0)
+        expect(resourceEvents[0]?.resource.status_code).toBe(200)
+      })
+
+    createTest('track both XHRs when the second wait for the respond of the first')
```

Comment:
was there an issue?

Reply:
I'm checking that the first request isn't canceled by the second one when waiting for it to be finished.